packages/hooks/feed/useFetchFeed.ts

```diff
@@ -19,8 +18,8 @@ import { extractGnoJSONString } from "@/utils/gno";
 
 export type PostsList = {
   list: Post[];
-  totalCount: number;
-} | null;
+  totalCount: number | undefined;
```

Comment:
is this the same as :
```
export type PostsList = {
  list: Post[];
  totalCount?: number;
}
```

Reply:
not entirely, `totalCount: number | undefined;` requires that the field is defined, `totalCount?: number;` does not